content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -113,34 +94,32 @@ stages:
         md5: ede2872bedbfe10342bb1c416e2f049f
 ```
 
-**Remember: DVC doesn’t replace Git.** Having codified data
-<abbr>artifacts</abbr> and [pipelines](/doc/start/data-pipelines), regular `git`
-commands are used to create versions (commits, tags, branches, etc.):
+> See [Data Pipelines](/doc/start/data-pipelines) to learn about stages.
 
-> Note that DVC automatically prevents Git from tracking data in the
-> <abbr>workspace</abbr> (via `.gitignore`).
+Having codified the data and models in the project, a regular `git` workflow can
+be used to create versions (commits), branches, etc. using the metafiles as
+proxies to the underlying data:
 
 ```dvc
-$ git add cleanup.sh
+$ git add training.py
 $ dvc add data/
 ...
 $ git add data.dvc data/.gitignore
+$ git commit -m 'First modeling experiment'
 
-$ git commit -m 'Data cleanup v1.0'
+# Iterate, repeat!
 ```
 
-> By matching versions of code with snapshots of the data at the time, we can
-> achieve full [reproducibility](/doc/start/data-pipelines).
+> Note that DVC prevents Git from tracking data in the workspace (via
+> `.gitignore`).
```

Comment:
Probably no need for this note but then the `git add data/.gitignore` is totally out of context. Should we remove both `data/.gitignore` in the `git add` line AND the note, or just the note? (Or keep both)

Reply:
I removed both (replaces with `...` above). Resolving